cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/TieFlowEq.java

```diff
@@ -21,6 +21,9 @@ public static void write(String id, String controlAreaId, String terminalId, Str
         CgmesExportUtil.writeStartId("TieFlow", id, false, cimNamespace, writer, context);
         CgmesExportUtil.writeReference("TieFlow.ControlArea", controlAreaId, cimNamespace, writer, context);
         CgmesExportUtil.writeReference("TieFlow.Terminal", terminalId, cimNamespace, writer, context);
+        writer.writeStartElement(cimNamespace, "TieFlow.positiveFlowIn");
+        writer.writeCharacters("true"); // always true
```

Comment:
About `positiveFlowIn`, it depends on the location of the terminal respect to the control area. For terminals of tie lines that are further away from the control area, this should be `true`. If the terminal is the near end of the tie line branch, it should be `false`. 

The typical configuration is that we have boundaries and the terminals of tie flows for control areas are defined at the boundary side, so for me it is ok to assume `true` by default. If we have to IGMs that share a boundary, and both define their tie flow at the boundary side of is own lines, both will set the `positiveFlowIn` to `true` in its tie flows. This can be seen in the conformity configuration MicroGrid BaseCase (https://github.com/powsybl/powsybl-core/tree/main/cgmes/cgmes-conformity/src/main/resources/conformity/cas-3-data-3.0.2/MicroGrid/BaseCase/MicroGrid-BaseCase-Merged)